### PR TITLE
Homebrew 2.6.0

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
Release post for Homebrew 2.6.0.

### CODE REVIEW

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- [`brew install --force-bottle` refuses to ever build from source](https://github.com/Homebrew/brew/pull/9064)
+- [`brew install` or `brew link` of a versioned keg-only formula will automatically unlink conflicted version formulae](https://github.com/Homebrew/brew/pull/9009)
+- [`brew shellenv` is significantly faster](https://github.com/Homebrew/brew/pull/8904)
+- [Homebrew has further reduced shallow clone usage (on request from GitHub)](https://github.com/Homebrew/brew/pull/8883)
+- [`brew linkage` and commands using the linkage cache have significantly better performance](https://github.com/Homebrew/brew/pull/8695)
+- [`brew bump-cask-pr` is a new developer command to create Homebrew/homebrew-cask pull requests](https://github.com/Homebrew/brew/pull/7986)
+
+Finally:
+
+- [Discourse](https://github.com/Homebrew/brew/pull/9144) and [IRC](https://github.com/Homebrew/brew/pull/8981) are now deprecated as official communication methods in favour of [GitHub Discussions.](https://github.com/Homebrew/brew/pull/8802).
```
Comment (bayandin) [inline _posts/2020-11-31-homebrew-2.6.0.md:33]:
```suggestion
- [Discourse](https://github.com/Homebrew/brew/pull/9144) and [IRC](https://github.com/Homebrew/brew/pull/8981) are now deprecated as official communication methods in favour of [GitHub Discussions](https://github.com/Homebrew/brew/pull/8802).
```

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
+- [`Homebrew.args` is deprecated](https://github.com/Homebrew/brew/pull/8795)
+- [All `Requirement`s are deprecated in Homebrew/core](https://github.com/Homebrew/brew/pull/9052)
+- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side).](https://github.com/Homebrew/brew/pull/9117)
+- [`brew tap-new` will setup GitHub Actions workflows to](https://github.com/Homebrew/brew/pull/8762) [upload to GitHub Releases](https://github.com/Homebrew/brew/pull/8410). Read [the blog post](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) for more documentation.
```
Comment (samford) [inline _posts/2020-11-31-homebrew-2.6.0.md:15]:
```suggestion
- [`brew tap-new` will set up GitHub Actions workflows to](https://github.com/Homebrew/brew/pull/8762) [upload to GitHub Releases](https://github.com/Homebrew/brew/pull/8410). Read [the blog post](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) for more documentation.
```

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- [GitHub deprecated their API's basic authentication](https://github.com/Homebrew/brew/pull/8826)
+
+Other changes since 2.5.0 I'd like to highlight are the following:
+
+- [`brew doctor` checks all tap branches, not just `homebrew-core`](https://github.com/Homebrew/brew/pull/9305)
```
Comment (SMillerDev) [inline _posts/2020-11-31-homebrew-2.6.0.md:20]: Is homebrew-core a branch in this?

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
+- [`Homebrew.args` is deprecated](https://github.com/Homebrew/brew/pull/8795)
+- [All `Requirement`s are deprecated in Homebrew/core](https://github.com/Homebrew/brew/pull/9052)
+- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side).](https://github.com/Homebrew/brew/pull/9117)
```
Comment (bayandin) [inline _posts/2020-11-31-homebrew-2.6.0.md:14]:
```suggestion
- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side)](https://github.com/Homebrew/brew/pull/9117).
```

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 1 above)
Comment (SMillerDev) [inline _posts/2020-11-31-homebrew-2.6.0.md:33]: Is this posted to discourse yet? I seem to recall the last message was "we're trying this other thing"

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 4 above)
Comment (sjackman) [reply _posts/2020-11-31-homebrew-2.6.0.md:14]:
```suggestion
- [macOS Homebrew running natively on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side)](https://github.com/Homebrew/brew/pull/9117). We currently recommend running Homebrew using Intel emulation with Rosetta 2.
```

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: 2.6.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
```
Comment (iMichka) [inline _posts/2020-11-31-homebrew-2.6.0.md:5]: `macOS Big Sur support on Intel`

I know that you talk about M1 at the end of the sentence, but as this is the first thing people will read, I would not like to raise wrong expectations.

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 3 above)
Comment (mistydemeo) [reply _posts/2020-11-31-homebrew-2.6.0.md:20]: The wording's slightly hard to parse without looking it up - I had to look at the PR to interpret what this one meant. How about this wording?
```suggestion
- [`brew doctor` checks the active branch for all taps, not just `homebrew-core`](https://github.com/Homebrew/brew/pull/9305)
```

**Review by mistydemeo — APPROVED** [review]
Left a comment, but otherwise looking good to me!

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 1 above)
Comment (MikeMcQuaid) [reply _posts/2020-11-31-homebrew-2.6.0.md:33]: Not posted but seems we’ve agreed on a direction. I’ll post an announcement tomorrow when I post this?

File: _posts/2020-11-31-homebrew-2.6.0.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
```
Comment (scpeters) [inline _posts/2020-11-31-homebrew-2.6.0.md:11]: I've noticed a few people reporting issues related to a change in `brew list` functionality that was deprecated in homebrew/brew#8851 (released in 2.5.7) and I believe it's intended to match the deprecation of `brew cask` commands. However, I believe the `odeprecated` calls from homebrew/brew#8851 were included in the conversions to `odisabled` in homebrew/brew#9209. I'm going to open a PR to narrowly un-`odisable` the changes from homebrew/brew#8851

Comment (scpeters) [reply _posts/2020-11-31-homebrew-2.6.0.md:11]: https://github.com/Homebrew/brew/pull/9357

**Comment by okuryu** [comment]
`2020-11-31-homebrew-2.6.0.md` would be correct to `2020-12-01-homebrew-2.6.0.md`.

**Comment by sjackman** [comment]
`2020-11-31` 😂 Time has no meaning in 2020. There are no rules anymore.

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 5 above)
Comment (MikeMcQuaid) [inline _posts/2020-11-31-homebrew-2.6.0.md:5]:
```suggestion
Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support on Intel, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
```

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 6 above)
Comment (MikeMcQuaid) [reply _posts/2020-11-31-homebrew-2.6.0.md:11]: Merged!

File: _posts/2020-11-31-homebrew-2.6.0.md (hunk 3 above)
Comment (MikeMcQuaid) [inline _posts/2020-11-31-homebrew-2.6.0.md:20]: The wording's slightly hard to parse without looking it up - I had to look at the PR to interpret what this one meant. How about this wording?
```suggestion
- [`brew doctor` checks the active branch for all taps, not just Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/9305)
```
